org.palladiosimulator.simulizar/schema/org.palladiosimulator.simulizar.interpreter.rdseffswitch.exsd

```diff
@@ -0,0 +1,102 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="org.palladiosimulator.simulizar" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="org.palladiosimulator.simulizar" id="org.palladiosimulator.simulizar.interpreter.rdseffswitch" name="RDSeff Switch Factory"/>
+      </appinfo>
+      <documentation>
+         Extension Point to add the interpretation of new PCM model elements to the RDSeffSwitch.
+      </documentation>
+   </annotation>
+
+   <element name="extension">
+      <annotation>
+         <appinfo>
+            <meta.element />
+         </appinfo>
+      </annotation>
+      <complexType>
+         <sequence minOccurs="1" maxOccurs="unbounded">
+            <element ref="rdseffswitch"/>
+         </sequence>
+         <attribute name="point" type="string" use="required">
+            <annotation>
+               <documentation>
+                  

```

Comment:
Please also add one or two sentences per <documentation></documentation> pair that I did not explicitly highlight in the review.